common/constants.js

```diff
@@ -385,5 +385,6 @@ module.exports = Object.freeze({
   },
   DOCKER_HOST_CONFIG: {
     PIDS_LIMIT: 150
-  }
+  },
+  SYSTEM_ERRORS: ['ECONNREFUSED', 'ECONNRESET', 'EPIPE', 'ETIMEDOUT', 'ESOCKETTIMEDOUT']
```

Comment:
Just a general question: in any case, the BOSH API status code is the driving factor for such errors. Do we need to handle these system errors as well? I know it's safer but just trying to understand-- if the response does not have a status code, that should be sufficient for a failure, isn't it?

Reply:
want to staggered on connection failure or down. For other cases wanted to fail, e.g. EADDRINUSE, EMFILE (source https://nodejs.org/api/errors.html#errors_class_systemerror). So, generic status code not found will lead to staggered in those cases also. Is that fine ?

Reply:
Sounds good...